vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -163,18 +164,8 @@ def save_container_service_port_configs_inventory(container_service, hashes, tar
     store_ids_for_new_records(container_service.container_service_port_configs, hashes, :ems_ref)
   end
 
-  def save_containers_inventory(container_group, hashes, target = nil)
-    return if hashes.nil?
-
-    container_group.containers(true)
-    deletes = if target.kind_of?(ExtManagementSystem)
-                container_group.containers.dup
-              else
-                []
-              end
-
-    save_inventory_multi(:containers, container_group, hashes, deletes, [:ems_ref])
-    store_ids_for_new_records(container_group.containers, hashes, :ems_ref)
+  def save_container_inventory(container_definition, hash, _target = nil)
```

Comment:
Since we probably want to maintain also the association with the `container_group` then we should add:

    hash[:container_group_id] = container_definition.container_group_id